### PR TITLE
Migrated sample to .NET 7.0 removing need for implicit flow

### PR DESCRIPTION
**Pull request by v-michaelmi**
## Purpose

* Update sample to use latest libraries and .NET 7.0 so that implicit flow is no longer needed.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Smoke test per instructions. No changes in behavior should be noted and you should no longer need implicit access tokens.

## What to Check
Verify that the following are valid
* Smoke test per instructions. No changes in behavior should be noted and you should no longer need implicit access tokens.

### CODE REVIEW

File: WebApp-graph-user/Call-MSGraph/AppCreationScripts/sample.json [hunk 1]
```diff
@@ -2,9 +2,10 @@
   "Sample": {
     "Title": "Enable your Blazor Single-page Application (SPA) to sign-in users with the Microsoft identity platform",
     "Level": 200,
-    "Client": "ASP .Net Core Blazor WebAssembly Web App",
+    "Client": "ASP.NET Core Blazor WebAssembly Web App",
     "RepositoryUrl": "ms-identity-blazor-wasm",
-    "Endpoint": "AAD v2.0"
+    "Endpoint": "AAD v2.0",
+    "Service": "Microsoft Graph"
```
Comment (kalyankrishna1) [inline WebApp-graph-user/Call-MSGraph/AppCreationScripts/sample.json:8]: please update additional properties required for code samples browser

File: WebApp-graph-user/Call-MSGraph/AppCreationScripts/sample.json [hunk 2]
```diff
@@ -14,17 +15,19 @@
     {
       "Id": "webApp",
       "Name": "WebApp-blazor-wasm",
-      "Kind": "WebApp",
+      "Kind": "SinglePageApplication",
+      "Sdk": "MsalNet",
       "Audience": "AzureADMyOrg",
-      "AllowImplicitFlow": true,
+      "AllowImplicitFlow": false,
```
Comment (derisen) [inline WebApp-graph-user/Call-MSGraph/AppCreationScripts/sample.json:21]:
```suggestion
```

Comment (derisen) [reply WebApp-graph-user/Call-MSGraph/AppCreationScripts/sample.json:21]: false by default

File: WebApp-graph-user/Call-MSGraph/AppCreationScripts/sample.json [hunk 3]
```diff
@@ -14,17 +15,19 @@
     {
       "Id": "webApp",
       "Name": "WebApp-blazor-wasm",
-      "Kind": "WebApp",
+      "Kind": "SinglePageApplication",
+      "Sdk": "MsalNet",
```
Comment (derisen) [inline WebApp-graph-user/Call-MSGraph/AppCreationScripts/sample.json:19]:
```suggestion
      "Sdk": "MsalJs",
```

File: WebApp-graph-user/Call-MSGraph/README.md [hunk 4]
```diff
@@ -1,65 +1,74 @@
...
-With respect to the previous chapter of the tutorial, this chapter adds the following steps:
-
-1. The client application acquires an Access Token for Microsoft Graph.
-1. The **Access Token** is used as a *bearer* token to authorize the user to call the [Microsoft Graph API](https://docs.microsoft.com/graph/overview)
-1. **Microsoft Graph API** responds with the resource that the user has access to.
+1. The client ASP.NET Core Blazor WebAssembly Web App uses the  to sign-in a user and obtain a JWT [ID Token](https://aka.ms/id-tokens) and an [Access Token](https://aka.ms/access-tokens) from **Azure AD**.
+1. The **access token** is used as a *bearer* token to authorize the user to call the Microsoft Graph protected by **Azure AD**.
 
-![Overview](./ReadmeFiles/topology.jpg)
+![Scenario Image](./ReadmeFiles/topology.png)
```
Comment (derisen) [inline WebApp-graph-user/Call-MSGraph/README.md:41]: looks like the image is busted

File: WebApp-graph-user/Call-MSGraph/README.md [hunk 5]
```diff
@@ -69,102 +78,77 @@ There is one project in this sample. To register it, you can:
...
-#### Step 2. Trust development certificates
-
-```console
-dotnet dev-certs https --clean
-dotnet dev-certs https --trust
-```
-
-Learn more about [HTTPS in .NET Core](https://docs.microsoft.com/aspnet/core/security/enforcing-ssl).
+1. Find the key `Enter_Application_ID_here` and replace the existing value with the application ID (clientId) of `WebApp-blazor-wasm` app copied from the Azure portal.
+1. Find the key `Enter_Tenant_ID_here` and replace the existing value with 'https://login.microsoftonline.com/'+$tenantId.
```
Comment (derisen) [inline WebApp-graph-user/Call-MSGraph/README.md:143]:
```suggestion
1. Find the key `Enter_Tenant_ID_here` and replace the existing value with your tenant ID.
```

**Review by derisen — APPROVED** [review]
LGTM

**Pull request merged**